project.owasp.yaml

```
@@ -0,0 +1,279 @@
+audience:
+  - builder
+  - defender
+community:
+  - name: proj-dependency-check
+    platform: slack
+    url: https://owasp.slack.com/archives/C2FTJJ26N
+    description: channel for the https://www.owasp.org/index.php/OWASP_Dependency_Check project - For general questions with the tool, check out the mailing
+      list https://groups.google.com/forum/#!forum/dependency-check. To report issues, please submit a GitHub issue.
+  - name: dependency-check
+    platform: slack
+    url: https://owasp.slack.com/archives/C082E5KB4HJ
+leaders:
+  - name: Jeremy Long
+    email: jeremy.long@owasp.org
+    github: jeremylong
+level: 4
+license:
+  - Apache-2.0
+  - EUPL-1.2
+  - GPL-3.0
+  - MIT
+name: OWASP Dependency-Check
+pitch: Dependency-Check is a Software Composition Analysis (SCA) tool suite that identifies project dependencies and checks if there are any known, publicly
+  disclosed, vulnerabilities.
+repositories:
+  - name: www-project-dependency-check
+    url: https://github.com/OWASP/www-project-dependency-check
+    description: OWASP Foundation Web Repository
+  - name: sbt-dependency-check
+    url: https://github.com/albuch/sbt-dependency-check
+    description: 'SBT Plugin for OWASP DependencyCheck. Monitor your dependencies and report if there are any publicly known vulnerabilities (e.g. CVEs).
+      :rainbow:'
+  - name: azuredevops
+    url: https://github.com/dependency-check/azuredevops
+    description: Dependency Check Azure DevOps Extension
+  - name: dependency-check-gradle
+    url: https://github.com/dependency-check/dependency-check-gradle
+    description: The dependency-check gradle plugin is a Software Composition Analysis (SCA) tool that allows projects to monitor dependent libraries for
+      known, published vulnerabilities.
+  - name: dependency-check-sonar-plugin
+    url: https://github.com/dependency-check/dependency-check-sonar-plugin
+    description: Integrates Dependency-Check reports into SonarQube
+  - name: Dependency-Check_Action
+    url: https://github.com/dependency-check/Dependency-Check_Action
+    description: Github action to run dependency check
+  - name: DependencyCheck
+    url: https://github.com/dependency-check/DependencyCheck
+    description: OWASP dependency-check is a software composition analysis utility that detects publicly disclosed vulnerabilities in application dependencies.
+  - name: DependencyCheck_Builder
+    url: https://github.com/dependency-check/DependencyCheck_Builder
+    description: It builds a dependency check image with the updated database every night
+  - name: DependencyCheck_Test
+    url: https://github.com/dependency-check/DependencyCheck_Test
+    description: Dependency check action test
+  - name: dependency-check-plugin
+    url: https://github.com/jenkinsci/dependency-check-plugin
+    description: Jenkins plugin for OWASP Dependency-Check. Inspects project components for known vulnerabilities (e.g. CVEs).
```

This one might be a grey area - probably needs for @jeremylong to weigh in on how "official" it is, since it's not within the dependency-check namespace. Probably depends if Jeremy has commit access to it.

---

While I believe I have access - for repositories not maintained by the core team, we likely need to mention this.
```suggestion
    url: https://github.com/jenkinsci/dependency-check-plugin
    description: Jenkins plugin for OWASP Dependency-Check. Inspects project components for known vulnerabilities (e.g. CVEs). This plugin is not maintained by the core dependency-check development team.
```